code/model/modelinterp.cpp

```diff
@@ -1765,6 +1780,80 @@ void parse_tmap(int offset, ubyte *bsp_data)
 	Parse_normal_problem_count += problem_count;
 }
 
+void parse_tmap2(int offset, ubyte* bsp_data)
+{
+	int pof_tex = w(bsp_data + offset + 24);
```

Comment:
This function appears to declare/define variables as they're needed, not the C style that the first two appear to be in.